README.md

```diff
@@ -0,0 +1,4 @@
+# Functions' Templates
+This is the monorepo for templates that are used in various OAuth functions
```

Comment:
Let's remove `OAuth`. I think this repo can be not only for OAuth functions.

Reply:
Done